src/components/forms/transaction/SplitField.tsx

```diff
@@ -69,6 +69,8 @@ export default function SplitField({
     form.trigger('splits');
   }, [value, exchangeRate, form, action, index]);
 
+  const defaultAccount = form.formState.defaultValues?.splits?.[index]?.fk_account as Account;
```

Comment:
use the `formState.defaultValues` instead of account watcher to narrow user options.